config/config.go

```diff
@@ -179,9 +178,24 @@ func Get() (Config, []FutureLog) {
 	}
 
 	var errs []FutureLog
-	config.ExternalIPV4, config.ExternalIPV6, errs = validateExternalIP(config.ExternalIP)
+	config.ExternalIPV4, config.ExternalIPV6, errs = validateExternalIP(config.ExternalIP, "SCREEGO_EXTERNAL_IP")
```

Comment:
I feel like the ipv4+6 doesn't have to be duplicated, what do you think of a config like:
```
type Config struct {
   	...
   	TurnExternal bool   `ignored:"true"`
	TurnIPV4     net.IP `ignored:"true"`
	TurnIPV6     net.IP `ignored:"true"`
	TurnPort     string `ignored:"true"`
}
```
With a validation like:
```go
var errs []FutureLog
if len(config.TurnExternalIP) > 0 {
    if len(config.ExternalIP) > 0 {
        logs = append(logs, futureFatal("SCREEGO_EXTERNAL_IP and SCREEGO_TURN_EXTERNAL_IP must not be both set"))
    }

    config.TurnIPV4, config.TurnIPV6, errs = validateExternalIP(config.TurnExternalIP, "SCREEGO_TURN_EXTERNAL_IP")
    config.TurnPort = config.TurnExternalPort
    config.TurnExternal = true
    logs = append(logs, errs...)
    if config.TurnExternalSecret == "" {
        logs = append(logs, futureFatal("SCREEGO_TURN_EXTERNAL_SECRET must be set if external TURN server is used"))
    }
} else if len(config.ExternalIP) > 0 {
    config.TurnIPV4, config.TurnIPV6, errs = validateExternalIP(config.ExternalIP, "SCREEGO_EXTERNAL_IP")
    logs = append(logs, errs...)
    split := strings.Split(config.TurnAddress, ":")
    config.TurnPort = split[len(split)-1]
} else {
    logs = append(logs, futureFatal("SCREEGO_EXTERNAL_IP or SCREEGO_TURN_EXTERNAL_IP must be set"))
}
```
TurnExternal can be used to decide if internal or external turn server should be used and the ipvX part doesn't have to be abstracted inside the turn.Server interface.

Reply:
this is a good suggestion, I like it!